app/src/test/java/com/google/samples/apps/sunflower/data/GardenPlantingTest.kt

```diff
@@ -26,8 +26,9 @@ import java.util.Calendar.YEAR
 
 class GardenPlantingTest {
 
-    @Test fun testDefaultValues() {
-        val gardenPlanting = GardenPlanting("1", "1")
+    @Test
+    fun testDefaultValues() {
+        val gardenPlanting = GardenPlanting("1").also { it.gardenPlantingId = 1 }
```

Comment:
Remove the `.also` block and add a test: `assertEquals(0L, gardenPlanting.gardenPlantingId)`

Reply:
Yes, you're right, the id should have to be given automatically.